### PR TITLE
Top donaters

### PR DESCRIPTION
**Pull request by Ben-21**

### CODE REVIEW

File: frontend/src/components/TopDonaters.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        for (let i = 0; i < string.length; i += 1) {
+            hash = string.charCodeAt(i) + ((hash << 5) - hash);
+        }
+        let color = '#';
+        for (let i = 0; i < 3; i += 1) {
+            const value = (hash >> (i * 8)) & 0xff;
+            color += `00${value.toString(16)}`.slice(-2);
+        }
+        return color;
+    }
```
Comment (WE-Kaito) [inline frontend/src/components/TopDonaters.tsx:38]: wow ok 👀

File: frontend/src/components/TopDonaters.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import {Project} from "../models/models.tsx";
+import styled from "@emotion/styled";
+import Avatar from '@mui/material/Avatar';
+
+type Props = {
+    project: Project;
+}
+
+export default function TopDonaters(props: Props) {
```
Comment (WE-Kaito) [inline frontend/src/components/TopDonaters.tsx:9]: typo: *Donators

**Review by WE-Kaito — APPROVED** [review]
LGTM! 🚀

File: frontend/src/components/TopDonaters.tsx (hunk 2 above)
Comment (Ben-21) [reply frontend/src/components/TopDonaters.tsx:9]: thx

**Pull request merged**